scripts/enable.sh

```diff
@@ -77,11 +51,11 @@ else
     compose_up="false"
 fi
 
-curl -L https://github.com/docker/compose/releases/download/1.1.0-rc2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+curl -L https://github.com/docker/fig/releases/download/1.1.0-rc2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
You're talking about the change from "compose" to "fig" right?  I have no idea how that changed.  I'll change it back.